src/nih_wayfinding/app/scripts/profiles/profile-service.js

```diff
@@ -83,6 +84,15 @@
         }
 
         /**
+         * Find current user profile in localstorage
+         *
+         * @return {object} The user object, deserialized
+         */
+        function fetchCurrentUserProfile() {
```

Comment:
Oh, that is the profile it's returning.  I thought it was maybe just the user name.